chromium/browser_type.go

```diff
@@ -124,16 +124,15 @@ func (b *BrowserType) initContext() context.Context {
 
 // Launch allocates a new Chrome browser process and returns a new api.Browser value,
 // which can be used for controlling the Chrome browser.
-func (b *BrowserType) Launch(opts goja.Value) api.Browser {
+func (b *BrowserType) Launch(opts ...goja.Value) api.Browser {
```

Comment:
Why does it need to be done in this way?

For example `browser.newContext()` can be used without supplying any options and it only takes a single `goga.Value` argument:

```go
func (b *Browser) NewContext(opts goja.Value) api.BrowserContext
```

Reply:
The browser context parse returns `ni`l if `opts` is `nil`:

```go
if opts != nil && !goja.IsUndefined(opts) && !goja.IsNull(opts) {
   ...
}
return nil
```

Couldn't we do something similar?

Reply:
That's a very good point 👍  I'll check it out, thanks.